include/hdmi_cec_driver.h

```diff
@@ -159,7 +159,7 @@ HDMI_CEC_STATUS HdmiCecOpen(int *handle);
  * @retval HDMI_CEC_IO_NOT_OPENED       - Module is not initialised
  * @retval HDMI_CEC_IO_INVALID_HANDLE   - An invalid handle argument has been passed
  *
- * @pre HdmiCecOpen() must be called before calling this API.
+ * @pre HdmiCecOpen(), HdmiCecRemoveLogicalAddress() must be called before calling this API.
```

Comment:
HdmiCecRemoveLogicAddresse is required only if  HdmiCecAddLogicAddress() is called , right ?